apollo-router/src/plugins/telemetry/apollo.rs

```diff
@@ -60,7 +60,7 @@ pub(crate) struct Config {
     #[serde(deserialize_with = "deserialize_header_name")]
     pub(crate) client_version_header: HeaderName,
 
-    /// The buffer size for sending traces to Apollo. Increase this if you are experiencing lost traces.
+    /// The buffer size for sending traces to Apollo. (deprecated, has no effect now)
     pub(crate) buffer_size: NonZeroUsize,
```

Comment:
should we remove it and create a migration ?

Reply:
I'll make a migration in https://github.com/apollographql/router/pull/2894 once I'm sure about the changes to sampling configuration